js/okex.js

```diff
@@ -3544,6 +3546,47 @@ module.exports = class okex extends Exchange {
         };
     }
 
+    async fetchBorrowRates (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetAccountInterestRate (params);
+        const timestamp = this.milliseconds ();
+        const data = this.safeValue (response, 'data');
+        const rates = [];
+        for (let i = 0; i < data.length; i++) {
+            const rate = data[i];
+            const currency = this.safeCurrencyCode (this.safeString (rate, 'ccy'));
+            rates[currency] = {
```

Comment:
If `rates` is a list, it cannot be used as an associative dictionary across languages. It has to be `const rates = {}` on line 3554.